sparse/impl/KokkosSparse_spmv_impl_merge.hpp

```diff
@@ -69,28 +69,28 @@ struct SpmvMergeHierarchical {
 
   template <bool NONZEROS_USE_SCRATCH, bool ROWENDS_USE_SCRATCH, bool Y_USE_SCRATCH, bool CONJ>
   struct SpmvMergeImplFunctor {
-    SpmvMergeImplFunctor(const y_value_type& _alpha, const AMatrix& _A, const XVector& _x, const YVector& _y,
+    SpmvMergeImplFunctor(const y_value_type& _alpha, const AMatrix& A, const XVector& _x, const YVector& _y,
                          const A_size_type pathLengthThreadChunk)
-        : alpha(_alpha), A(_A), x(_x), y(_y), pathLengthThreadChunk_(pathLengthThreadChunk) {}
+        : alpha(_alpha), A_(A), x(_x), y(_y), pathLengthThreadChunk_(pathLengthThreadChunk) {}
 
     y_value_type alpha;
-    AMatrix A;
+    AMatrix A_;
```

Comment:
Then this can be kept as `A` which might be easier to work with in the implementation below?